tooling/eslint/package.json

```diff
@@ -8,9 +8,8 @@
     "@eslint/css": "^0.8.1",
     "@html-eslint/eslint-plugin": "^0.39.0",
     "@html-eslint/parser": "^0.39.0",
-    "browserslist-config-baseline": "^0.4.0",
```

Comment:
Quick note: removing compat also removes the `browserslist-config-baseline` example. Would you like me to add a separate compat demo or restore a commented compat block here?